pkg/xds/envoy/listeners/v3/timeout_configurer.go

```diff
@@ -18,10 +18,6 @@ type TimeoutConfigurer struct {
 }
 
 func (c *TimeoutConfigurer) Configure(filterChain *envoy_listener.FilterChain) error {
-	if c.Conf == nil {
-		return nil
-	}
-
```

Comment:
Why we are removing this check? Was it unnecessary?

Reply:
Empty conf works well to set all zeros, and I'm using it [here](https://github.com/kumahq/kuma/pull/4365/files#diff-8a0428ca30722cd8c5ae3a9f3842e6cb51b71d2b5f423d4ab838f50b2ea094b0R121)

Reply:
got it!